wagtailio/core/blocks.py

```diff
@@ -226,10 +231,24 @@ def get_context(self, value, parent_context=None):
             context["value"]["url"] = value["cta_page"].get_url
         return context
 
+
+class CTALinkStructValue(blocks.StructValue):
+    def url(self):
+        cta_url = self.get("cta_url")
+        cta_page = self.get("cta_page")
+        return cta_url or cta_page.url
+
+
+class CTABlock(CTALinkMixin, blocks.StructBlock):
```

Comment:
Note: Something to be careful of is the Python MRO(Method Resolution Order). As the MRO is left to right, we're inheriting a modified StructBlock(The mixin) and then inheriting the `StructBlock` again from a direct import, we only need the mixin. Take a look at this explainer when you can https://www.geeksforgeeks.org/method-resolution-order-in-python-inheritance/

Reply:
Ah! I see! Many thanks for this :+1:

Reply:
Fixed in 9d75fc4